### PR TITLE
Update 4.1.3

### PR DESCRIPTION
**Pull request by SC426**
[Jira](https://anaconda.atlassian.net/browse/PKG-1226)
[Upstream](https://github.com/getmoto/moto)
[Changelog](https://github.com/getmoto/moto/blob/master/CHANGELOG.md)

### Actions
- Update version number and sha256
- Update dependencies
- Linter fixes

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
There seems to be a change in the `dev_url` from 

https://github.com/spulec/moto

to 

https://github.com/getmoto/moto

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,33 +24,29 @@ requirements:
...
-    - pytz
     - python-dateutil >=2.1,<3.0.0
-    - python-jose >=3.1.0,<4.0.0
-    - docker-py >=2.5.1
     - responses >=0.13.0
-    - MarkupSafe !=2.0.0a1
-    - cfn-lint >=0.4.0
-    - flask
+    - jinja2 >=2.10.1
+    - importlib_metadata  # [py<38]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:37]: We do not build for python <3.8 so you can remove `- importlib_metadata  # [py<38]`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -24,33 +24,29 @@ requirements:
...
+    - python-jose[cryptography] >=3.1.0,<4.0.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:38]: The correct is `python-jose >=3.1.0,<4.0.0`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -24,33 +24,29 @@ requirements:
...
+    - ecdsa !=0.15
+    - docker-py >=2.5.1
+    - graphql-core
+    - pyyaml >=5.1
+    - cfn-lint >=0.40.0
+    - sshpubkeys >=3.1.0
+    - flask !=2.2.0,!=2.2.1
     - flask_cors
     - jsondiff >=1.1.2
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:47]: `pyparsing>=3.0.7` is missing, see https://github.com/getmoto/moto/blob/642ef7841cf7833d26bd21dd5eaab064b19eee69/setup.cfg#LL53C5-L53C21

**Review by cbouss — APPROVED** [review]
LGTM

**Pull request merged**